app/component/map/popups/DynamicParkingLotsPopup.js

```diff
@@ -35,6 +37,50 @@ class DynamicParkingLotsPopup extends React.Component {
     lon: PropTypes.number.isRequired,
   };
 
+  getPaidHours() {
+    const NOW = new Date();
```

Comment:
Most important are opening hours. For paid hours, we don't need to evaluate if we are "open", but just display them.